src/v/kafka/server/handlers/fetch.h

```diff
@@ -297,13 +288,18 @@ struct read_result {
 // struct aggregating fetch requests and corresponding response iterators for
 // the same shard
 struct shard_fetch {
+    explicit shard_fetch(op_context::latency_point start_time)
+      : start_time{start_time} {}
```

Comment:
Changes to this struct largely related to replacing "metrics" (auto_measurement) with a time point.